hyperspy/misc/axis_tools.py

```diff
@@ -17,7 +17,7 @@
 
 
 import numpy as np
-from pint.unit import Unit
+from pint import Unit
```

Comment:
To be consistent and potentially avoid future surprise, it would be better to use `Unit` from the unit registry instance used in hyperspy. It means that `_ureg` needs to be imported here (`from hyperspy.api_nogui import _ureg`) and then `_ureg.Unit` needs to be used.